package.json

```diff
@@ -7,7 +7,7 @@
     "JSON2": "0.1.0",
     "batch": "0.5.0",
     "browserify": "13.0.0",
-    "browserify-istanbul": "0.1.5",
+    "browserify-istanbul": "^2.0.0",
```

Comment:
this doesn't follow the pattern of the other dependencies...be mindful to be consistent